client/coral-framework/graphql/mutations/index.js

```diff
@@ -171,3 +172,22 @@ export const stopIgnoringUser = graphql(STOP_IGNORING_USER, {
     };
   }
 });
+
+export const editComment = graphql(EDIT_COMMENT, {
+  props: ({mutate}) => {
+    return {
+      editComment: (id, asset_id, edit) => {
+        return mutate({
+          variables: {
+            id,
+            asset_id,
+            edit,
+          },
+          refetchQueries: [
+            'EmbedQuery'
```

Comment:
Why are you refetching the `EmbedQuery`? We don't want to rely on refetching for mutations.